local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Right, this is what I was thinking as well. We were always thinking of using RDS or equivalent for production, but those are the advantages of using a Docker-based DB (for developers and other researchers).